src/Eto.Wpf/Forms/Controls/DrawableHandler.cs

```diff
@@ -163,18 +168,29 @@ public override void OnUnLoad(EventArgs e)
 			UnRegisterScrollable();
 		}
 
-		public void Create()
+		protected override TControl CreateControl()
 		{
-			Control = new EtoMainCanvas
+			return new EtoMainCanvas
 			{
 				Handler = this,
 				SnapsToDevicePixels = true,
 				FocusVisualStyle = null,
 				Background = swm.Brushes.Transparent
-			};
+			} as TControl;
+		}
+
+		protected override void Initialize()
+		{
+			base.Initialize();
+
 			Control.Loaded += Control_Loaded;
 		}
 
+		public void Create()
+		{
+			CreateControl();
```

Comment:
This line should be removed, CreateControl is called automatically.

Reply:
Thanks for the comment, Applied.

Reply:
Great, thank you!